src/main/java/com/shippingcostestimator/enterprise/dto/Destination.java

```diff
@@ -12,6 +12,6 @@
  */
 public @Data
 class Destination {
-    String startAddress;
-    String endAddress;
+    private String startAddress;
+    private String endAddress;
```

Comment:
Added private to the destination and shipping cost class variables for best practices

Reply:
Good.